src/main/java/racingcar/domain/car/Cars.java

```diff
@@ -52,16 +39,16 @@ public void add(Car car) {
         cars.add(car);
     }
 
-    public void moveAll() {
+    public void moveAll(RandomGeneratable randomGeneratable) {
         cars.forEach((car) -> car.move(
-                RandomGenerator.createRandom(RANDOM_MIN_BOUND, RANDOM_MAX_BOUND))
+                randomGeneratable.generateNumber(RANDOM_MIN_BOUND, RANDOM_MAX_BOUND))
         );
     }
 
-    public Names getWinnersNames() {
-        return Names.from(getCarsByPosition(getFirstPosition()).stream()
-                .map(Car::getName)
-                .collect(Collectors.toList()));
+    public List<Name> getWinnersNames() {
+        return getCarsByPosition(getFirstPosition()).stream()
+                .map(car -> car.getName())
```

Comment:
```suggestion
                .map(Car::getName)
```
메서드 레퍼런스를 사용해도 좋을 것 같아요 !

Reply:
적용하였습니다 😃